_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -2956,6 +2985,111 @@ private static function core_get_plugins() {
 		return array();
 	}
 
+
+	/**
+	 * Get third party plugin API keys.
+	 *
+	 * @param WP_REST_Request $request {
+	 *     Array of parameters received by request.
+	 *
+	 *     @type string $slug Plugin slug with the syntax 'plugin-directory/plugin-main-file.php'.
+	 * }
+	 */
+	public static function get_api_key( $request ) {
```

Comment:
This is a bit more nit picky. I think it would be cleared if we renamed `get_api_key`  to `get_service_api_key` and the same for update and delete methods.

Reply:
Changed this for all related functions (`get_service_api_key` | `update_service_api_key` | `delete_service_api_key` | `validate_service_api_key` | `validate_service_api_key_googlemaps`) in https://github.com/Automattic/jetpack/pull/10393/commits/8274b5e211418496252ea6a58c6882d06c9c5ba9